### DISCUSSION TITLE
Payment Gateway Integration

### DISCUSSION THREAD

**Discussion by sanjeevbrok99** [payloadcms/payload#1426] [Q&A] [answered]
Can we integrate Payment Gateway Api in Payload????

**Comment by jacobsfletch** [comment, marked as answer]
@sanjeevbrok99 yes you should be able to integrate with _any_ third party service, including payment gateways. Stripe is especially good at this, but the integration would look largely the same for any service. Just use the Payload hooks to trigger your payment events: https://payloadcms.com/docs/hooks/overview#hooks-overview
